packages/core/src/feedback.ts

```diff
@@ -0,0 +1,61 @@
+import type { EventHint, FeedbackEvent, SendFeedbackParams } from '@sentry/types';
+import { dropUndefinedKeys } from '@sentry/utils';
+import { getClient, getCurrentScope } from './currentScopes';
+import { createAttachmentEnvelope } from './envelope';
+
+/**
+ * Send user feedback to Sentry.
+ */
+export function captureFeedback(
+  feedbackParams: SendFeedbackParams,
+  hint?: EventHint & { includeReplay?: boolean },
+): string {
+  const { message, name, email, url, source, attachments, associatedEventId } = feedbackParams;
+
+  const client = getClient();
+  const transport = client && client.getTransport();
+  const dsn = client && client.getDsn();
+
+  const feedbackEvent: FeedbackEvent = {
+    contexts: {
+      feedback: dropUndefinedKeys({
+        contact_email: email,
+        name,
+        message,
+        url,
+        source,
+        associated_event_id: associatedEventId,
+      }),
+    },
+    type: 'feedback',
+    level: 'info',
+  };
+
+  if (client) {
+    client.emit('beforeSendFeedback', feedbackEvent, hint);
+  }
+
+  const eventId = getCurrentScope().captureEvent(feedbackEvent, hint);
+
+  // For now, we have to send attachments manually in a separate envelope
+  // Because we do not support attachments in the feedback envelope
+  // Once the Sentry API properly supports this, we can get rid of this and send it through the event envelope
+  if (client && transport && dsn && attachments && attachments.length) {
```

Comment:
@cmanallen @aliu3ntry Is this still true?

Reply:
You can send whatever you want in an envelope.  You're just capped to three items.  At least, as far as I know.

Reply:
I think Andrew has a PR up but it's not merged yet

Reply:
I'm confused on how that PR solves the problem?  Are you trying to put attachments into the feedback envelope item?  Or are you trying to send attachments in the same envelope and a feedback item?  These are two different problems.

Let's talk about this on Monday before my PTO.

Reply:
Confirmed with @c298lee -- same envelope as feedback item. Format will look like
<headers>
<feedback item>
<attachment item>
<attachment item>
...

For screenshots, there will be at most 1 attachment item

Reply:
See https://github.com/getsentry/team-replay/issues/393

Reply:
I discussed this with Andrew, but we want to send attachments in the same envelope as the feedback, the attachment will be its own item. We want to use `getCurrentScope().addAttachment()` to add the attachment to the envelope.

Reply:
OK, to clarify: This means I will remove the code to add attachments to a separate envelope, and also remove the `attachments` option for the method, instead expecting this to be added to the scope the same way as for any other event? --> This will reduce bundle size and complexity here too, because this should be handled OOTB by `captureEvent` so this should "just work" really. 

This is OK and will work (with up-to-date self-hosted, at least) today?

Reply:
It looks like https://github.com/getsentry/team-replay/issues/393 hasn't landed yet; but @mydea the SDK should send attachments and feedback together IMO.
Why?
- Sending them in the same payload gives us better guarantees that both parts made it successfully, so we won't get into cases where random attachments are ingested, but no matching feedback exists.
- We'll save a bunch of SDK code
- We can take advantage of being in alpha & beta state for the screenshot feature specifically.... I think it's ok to have a higher min-requirement on the server because turning on screenshots is opt-in right now.

For example, in the docs we could say something like (this is the idea, not the verbatim docs i'd write):
> v8 of the Feedback SDK supports allowing users to include a screenshot with their feedback submission.
> On-Prem users must update to version _${after getsentry/team-replay#393 lands}_ before enabling `showScreenshot: true` in the SDK config.

Reply:
Yeah, I agree that this makes sense, we just need to know how long it will take to land this and when it will be on-prem, and decide if we want to hold off on merging this until we know this!